character-physics.js

```diff
@@ -63,14 +66,26 @@ class CharacterPhysics {
     timeDiffS,
   ) {
     if (this.player.avatar && physicsManager.physicsEnabled) {
+
+      this.rb = this.player.avatar.app.physicsObjects[0];
+      //this.debugCapsule = this.player.avatar.app.debugCapsule;;
+
+      if(!this.rb) return;
+
+      physicsManager.disableGeometryQueries(this.rb);
+      physicsManager.enablePhysicsObject(this.rb);
       // capsule physics
       if (!this.player.hasAction('sit')) {
-        applyVelocity(this.player.position, this.velocity, timeDiffS);
+        //applyVelocity(this.player.position, this.velocity, timeDiffS);
+
 
-        this.player.updateMatrixWorld();
-        this.player.matrixWorld.decompose(localVector, localQuaternion, localVector2);
+        this.rb.updateMatrixWorld();
+        this.rb.matrixWorld.decompose(localVector, localQuaternion, localVector2);
+        localQuaternion.copy(this.player.quaternion);
 
-        const collision = this.collideCapsule(localVector, localQuaternion2.set(0, 0, 0, 1));
+        const capsuleOffset = new THREE.Vector3(0, this.player.avatar.height/2, 0);
+        localVector.add(capsuleOffset);
+        const collision = this.collideCapsule(localVector, localQuaternion2.set(0, 0, 0, 1)); // TODO: Replace with physicsManager.isGrounded restitution check

```

Comment:
We should not collide physics here. This should be removed.

Reply:
I also don't think `isGrounded` is something you can ask after the simulation.

Grounded state must come from the simulation. After that, we don't know if we are truly grounded since the state has already been restituted and that knowledge has been lost.